dart/lib/src/sentry_options.dart

```diff
@@ -133,7 +133,7 @@ class SentryOptions {
   String? serverName;
 
   /// Sdk object that contains the Sentry Client Name and its version
-  SdkVersion sdk = SdkVersion(name: sdkName, version: sdkVersion);
+  SdkVersion sdk = SdkVersion(name: sdkName(runsOnWeb), version: sdkVersion);
```

Comment:
can we use platformChecker.isWeb

Reply:
Yeah, we would need to set this in constructor.